src/pages/info/[slug].js

```diff
@@ -0,0 +1,141 @@
+import {
+  Box,
+
+  Button, Card,
+
+  CardActions, CardMedia,
+
+  IconButton,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import LinkUi from '@material-ui/core/Link';
+import { ArrowBack } from '@material-ui/icons';
+import { useRouter } from 'next/router';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { useAuthState } from 'react-firebase-hooks/auth';
+import Head from '../../components/Head';
+import firebase from '../../config/firebase';
+import { axiosGet } from '../../shared/axios';
+import isBrowser from '../../shared/browserCheck';
+import richTextHelper from '../../shared/rich-text';
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    borderRadius: 0,
+    margin: 0,
+    backgroundColor: theme.palette.info.main,
+    color: theme.palette.secondary.dark,
+  },
+  content: {
+    backgroundColor: theme.palette.secondary.dark,
+    color: theme.palette.primary.light,
+  },
+  button: {
+    margin: '0 auto',
+  },
+}));
+
+const InfoPage = ({ propContent, previewMode }) => {
+  const router = useRouter();
+  const classes = useStyles();
+  const [user] = isBrowser ? useAuthState(firebase.auth()) : [{}];
+  const [content, setContent] = useState(propContent);
+  const { slug } = router.query;
+
+  useEffect(() => {
+    if (previewMode && user) {
+      axiosGet(`pages/${slug}`, {
+        headers: {
+          'X-PREVIEW-MODE': 'preview',
+          authorization: `Bearer ${user.xa}`,
```

Comment:
`user.xa` doesn't work anymore – you'll need to `getIdToken` (like in other places).